server/src/main/scala/de/frosner/broccoli/controllers/InstanceController.scala

```diff
@@ -132,7 +134,7 @@ object InstanceController {
         .right[InstanceError, Account](loggedIn)
         .ensure(InstanceError.RolesRequired(Role.Administrator): InstanceError)(_.role == Role.Administrator)
       _ <- Either
-        .fromOption(instanceCreation.parameters.get("id"), InstanceError.IdMissing: InstanceError)
```

Comment:
There should be `EitherT.fromTry` or?

Reply:
I used `toOption` on the Try object but you are right I am squeezing two different errors into a single one. Will split it into two.